### DISCUSSION TITLE
How to make the bot move 1 block?

### DISCUSSION THREAD

**Discussion by Fluiding** [PrismarineJS/mineflayer#2041] [General]
## Is your feature request related to a problem? Please describe.
Yes, I cannot make the bot move. I researched about an hour but still couldn't figure it out.

## Describe the solution you'd like
A code that fixes the problem + an explanation of why it doesn't work.

## Describe alternatives you've considered
I've tried using mineflayer-navigate, watching tutorials but none of them match what I am trying to do.

## Additional context
None

**Comment by TheBlueBurger** [comment]
Please join the [discord server](https://discord.gg/ERZWEVmj) for support

**Comment by aesthetic0001** [comment]
or GH discussions
